include/cell.tcc

```diff
@@ -648,9 +661,9 @@ inline Eigen::Matrix<double, Tdim, 1>
   // Check if the first trial xi is just outside the box
   for (unsigned i = 0; i < nr_xi.size(); ++i) {
     if (nr_xi(i) < -1. && nr_xi(i) > -1.001)
```

Comment:
@kks32 Do you want the if check to be modified to be -1 + machepsilon too?

Reply:
I feel that we need to check for bigger actually, as the inaccuracy springs from affine transformation. However, I am not sure 1E-3 is the magic number.

Reply:
I used 1E-3 because if the material point is really outside, NR would converge outside - so no problem. Should we have a variation on this?